masterlist.yaml

```
@@ -7110,6 +7110,22 @@ plugins:
         util: '[TES4Edit v4.0.3h (Hotfix 1)](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 6
 
+  - name: 'Daedric Lord( Armor|-Classic Bound Armor)\.esp'
```

The regex can be simplified a bit.

```suggestion
  - name: 'Daedric Lord(-Classic Bound)? Armor\.esp'
```